src/components/Profile/Settings/NotificationsComponent/Notification.js

```diff
@@ -56,14 +91,22 @@ export class Notification extends Component {
 }
 Notification.propTypes = {
   notification: PropTypes.object.isRequired,
-  createOne: PropTypes.func.isRequired,
+  createConfiguration: PropTypes.func.isRequired,
+  updateOne: PropTypes.func.isRequired,
+  getNotificationConfiguration: PropTypes.func.isRequired,
   errorMessage: PropTypes.string
 };
 const mapStateToProps = ({ notificationReducer }) => ({
   notification: notificationReducer.config,
```

Comment:
@MBenedicte, as you're dealing with notification configuration, the logic key name would be configuration not notification, something like this:
```suggestion
  configuration: notificationReducer.config,
```